README.md

```diff
@@ -30,6 +29,17 @@ $ cmake --install build
 
 If you want to build libkeyfinder statically, add `-DBUILD_SHARED_LIBS=OFF` to the first call to `cmake` above.
 
+### MacOS Installation
```

Comment:
Thanks for working on this. I'd prefer to not have a separate section for different operating systems as it's pretty much the same for each OS. Instead of adding a new section, how about adding a link to https://brew.sh/ on the old

```
* macOS and homebrew: `$ brew install fftw catch2`
```
line

and adding a sentence that `/usr/local` is a typical location to install to for macOS?

Reply:
You can add `cmake` to the package list if you want too. I suppose that wouldn't hurt for the Linux distro package lists either. That's not needed for vcpkg on Windows because vcpkg requires cmake already.

Reply:
Updated as requested. How does that look?